search/tasks.py

```diff
@@ -41,8 +44,8 @@ def post_indexing_handler(program_enrollments):
     for program_enrollment in program_enrollments:
         _send_automatic_emails(program_enrollment)
 
-        if feature_sync_user:
-            _sync_user_to_channels(program_enrollment.user.id)
+        # only update for discussion queries for now
```

Comment:
Are we still checking the feature flag somewhere?

Reply:
Yup, it's done at the start of the `discussions.tasks.sync_channel_memberships` task now.